testsuite/python/reaction_methods.py

```diff
@@ -148,17 +148,17 @@ def check_reaction_parameters(reactions, parameters):
     def test_interface(self):
         # reaction ensemble
         method = espressomd.reaction_ensemble.ReactionEnsemble(
-            kT=1.5, exclusion_radius=0.8, seed=12)
-        self.check_interface(method, kT=1.5, exclusion_radius=0.8, gamma=1.2)
+            kT=1.5, exclusion_range=0.8, seed=12)
+        self.check_interface(method, kT=1.5, exclusion_range=0.8, gamma=1.2)
 
         # constant pH ensemble
         method = espressomd.reaction_ensemble.ConstantpHEnsemble(
-            kT=1.5, exclusion_radius=0.8, seed=12, constant_pH=10)
-        self.check_interface(method, kT=1.5, exclusion_radius=0.8, gamma=1.2)
+            kT=1.5, exclusion_range=0.8, seed=12, constant_pH=10)
+        self.check_interface(method, kT=1.5, exclusion_range=0.8, gamma=1.2)
 
         # Widom insertion
         method = espressomd.reaction_ensemble.WidomInsertion(kT=1.6, seed=12)
-        self.check_interface(method, kT=1.6, exclusion_radius=0., gamma=1.)
+        self.check_interface(method, kT=1.6, exclusion_range=0., gamma=1.)
```

Comment:
consider adding an argument `exclusion_radius_per_type={1: 0.1}` in each class and `check_interface` call, and then after line 73 adding the following:
```python
        self.assertEqual(list(method.exclusion_radius_per_type.keys()), [1])
        self.assertAlmostEqual(
            method.exclusion_radius_per_type[1],
            exclusion_radius_per_type[1],
            delta=1e-10)
        method.exclusion_radius_per_type = {2: 0.2}
        self.assertEqual(list(method.exclusion_radius_per_type.keys()), [2])
        self.assertAlmostEqual(
            method.exclusion_radius_per_type[2],
            0.2,
            delta=1e-10)
```
This will provide code coverage for the setters and getters and guarantee they have been properly implemented.

The goal of `self.assertEqual(list(method.exclusion_radius_per_type.keys()), [2])` is to make sure the key exists before we check the corresponding exclusion radius with ` self.assertAlmostEqual()` (if the key is not in the dict, we want the test case to halt with a meaningful error message, instead of halting the entire testsuite with a `KeyError` exception).